buildUtilities/releaseTask.m

```diff
@@ -0,0 +1,63 @@
+function [toolboxOptions] = releaseTask(toolboxVersion)
+%GENERATETOOLBOX Function that generates a toolbox for the boss device API
+
+arguments
+    toolboxVersion (1,:) {mustBeText} = '0.0'
+end
+
+% Toolbox Parameter Configuration
+
+toolboxFolder = "../../bossdevice-api-matlab";
+identifier = "bossdevice-api-matlab";
```

Comment:
As discussed please consider using properties of the project object whenever possible. If information is still empty, please add the required information to the project details too.